templates/model.txt

```diff
@@ -11,6 +15,8 @@ module.exports = mongoose.model('<%= table %>', schema, '<%= table %>');
 <% } else {
 %>module.exports = (sequelize, DataTypes) => {
   const { Sequelize } = sequelize;
+  // This section contains columns mapping.
```

Comment:
```suggestion
  // This section contains the fields of your model and map them to the column of your table.
```

Reply:
This is just a suggestion. Tell me what you think ?

Reply:
SGTM. ping @marcperrinp

Reply:
This sections contains the fields of your model, mapped to your table's columns. Learn more here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/models/enrich-your-models#declaring-a-new-field-in-a-model

Reply:
Well when I read a comments I prefer it to be visible fully without need to scroll. So here your comment will be too long and so I think it's great on 2 lines